bittide/src/Bittide/Transceiver/Comma.hs

```diff
@@ -0,0 +1,49 @@
+-- SPDX-FileCopyrightText: 2024 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+module Bittide.Transceiver.Comma where
+
+import Clash.Explicit.Prelude
+
+import Bittide.Arithmetic.Time (IndexMs)
+import Bittide.SharedTypes (Bytes, Byte)
+import Clash.Class.Counter (Counter(countSuccOverflow))
+
+-- | Number of milliseconds to generate commas for
+defTimeout :: SNat 1
+defTimeout = SNat
```

Comment:
I don't see the benefit of offering a default value at this point, as fixing a particular value has no benefit against any other value here. Passing just `d1` to `generator` at every usage looks more reusable and shorter to me. If a value shared among multiple instances is required it still can be declared and knowing that the natural corresponds to milliseconds still requires looking at the documentation anyway. Also the detour via the term level feels kind of strange.

Reply:
That's fair. The thing I was doing here was purely moving it to _milliseconds_, which makes the trick to run simulations run faster actually work. I'll add a TODO comment.

Reply:
I think you did not get my point. My proposal is to completely remove `defTimeout` from this module and replace every usage of it in the remaining code with `d1`. Is there any particular argument against this change?

Reply:
It's only used in one place, so no, I can remove it. I could even make it entirely unconfigurable to begin with.